### PR TITLE
Cursr

### PR DESCRIPTION
**Pull request by dfberger**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

Note: cursr.app (https://cursr.app/) host releases on github, but has no github code repository, hence the audit complains:

```
audit for cursr: failed
 - GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

submitting for inclusion as an exception.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully. 
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/cursr.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+cask "cursr" do
+  arch arm: "mac-arm64", intel: "mac-x64"
```
Comment (SMillerDev) [inline Casks/cursr.rb:2]:
```suggestion
  arch arm: "arm64", intel: "x64"
```

File: Casks/cursr.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  version "1.4.11"
+  sha256 arm:   "4552ea31b58f2b370c8891aae4bbf164095ddef1cffcc2507440fe0c331dcaed",
+         intel: "5ed9db1eeee7697741c061333b344006190d83264e5bb2dd6096b35438c913ee"
+
+  url "https://github.com/bitgapp/Cursr/releases/download/v#{version}/Cursr-#{arch}.dmg",
```
Comment (SMillerDev) [inline Casks/cursr.rb:8]:
```suggestion
  url "https://github.com/bitgapp/Cursr/releases/download/v#{version}/Cursr-mac-#{arch}.dmg",
```

**Comment by miccal** [comment]
Thank you for your submission @dfberger, but we cannot accept it at this time because it does not meet our [minimum notability requirements](https://docs.brew.sh/Acceptable-Casks#rejected-casks):

> + The Cask is for an app that is too obscure. Examples:
>   + An app from a code repository that is not notable enough (under 30 forks, 30 watchers, 75 stars).

Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if it gets notable enough we would then be happy to consider adding this to `homebrew-cask`.

**Comment by straxhaber** [comment]
Note to anyone who comes to this page: make sure to fork, watch, and star the `cursr` repository so we can get this added to Homebrew :) Awesome tool! I came here looking for a replacement for Wraparound, which is defunct.
https://github.com/bitgapp/Cursr

**Comment by straxhaber** [comment]
Thank you @bevanjkay <3. It looks like all of the checks passed besides the notability check that miccal previously mentioned.

**Comment by bevanjkay** [comment]
@straxhaber there's a couple of small changes that need to made. Which I can add more detail to soon. But we should be able to ignore the Notability audit as the repository only hosts the binaries so won't receive stars and forks in line with notability.

**Comment by bevanjkay** [comment]
Thank you @dfberger

**Review by bevanjkay — APPROVED** [review]

**Comment by straxhaber** [comment]
Thank you @bevanjkay, @dfberger, and @SMillerDev!

**Pull request merged**